### PR TITLE
Adds Getting Started, Projects sections to README

### PR DESCRIPTION
**Pull request by cybtachyon**
docs: (README.MD)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +102,12 @@ A high level breakdown of the components of the framework:
  * Download release and development [installers and packages](http://www.monogame.net/downloads/).
  * Follow [@MonoGameTeam](https://twitter.com/monogameteam) on Twitter.
 
+## Projects using MonoGame
```
Comment (tomspilman) [inline README.md]: I would prefer we don't start maintaining a list like this in the README.

Comment (cybtachyon) [reply README.md]: Where would you recommend one be maintained then? A separate repo?

Comment (Apostolique) [reply README.md]: A list like that is maintained in https://github.com/aloisdeniel/awesome-monogame

Comment (cybtachyon) [reply README.md]: Oh sweet, thanks! Updating the PR.

**Comment by cybtachyon** [comment]
@tomspilman Updated the PR to just link to the community-maintained projects list.

File: README.md (hunk 1 above)
Comment (cybtachyon) [reply README.md]: Can we mark this resolved and get this merged?

Comment (harry-cpp) [reply README.md:105]: Tom meant "Projects using MonoGame" shouldn't be in the README at all.

**Comment by tomspilman** [comment]
We don't need this.

**Pull request closed without merging**